micromasters_import/management/commands/queries/002_courserun_enrollment.sql

```diff
@@ -0,0 +1,65 @@
+--courserun migration from MicroMaster to MITxOnline
+INSERT INTO public.courses_courserun(
+    title,
+    courseware_id,
+    run_tag,
+    courseware_url_path,
+    start_date,
+    end_date,
+    enrollment_start,
+    enrollment_end,
+    expiration_date,
+    live,
+    created_on,
+    updated_on,
+    course_id)
+SELECT
+    mm_courserun.title,
+    mm_courserun.edx_course_key,
+    LEFT(CASE WHEN position('+' in mm_courserun.edx_course_key) > 0
+    THEN (regexp_match(mm_courserun.edx_course_key, '[^+]+$'))[1]
+    ELSE (regexp_match(mm_courserun.edx_course_key, '[^/]+$'))[1] END, 10) AS run_tag,
+    mm_courserun.enrollment_url,
+    mm_courserun.start_date,
+    mm_courserun.end_date,
+    mm_courserun.enrollment_start,
+    mm_courserun.enrollment_end,
+    mm_courserun.enrollment_end + INTERVAL '1 day',
```

Comment:
If expiration dates are used to restrict access to a closed course, we should leave them blank instead of generating an expiration date. Unlike xPRO courses, where access is restricted after some time, access to mitxonline courses is perpetual.

Reply:
I believe `expiration_date` is used to determine when we stop syncing course run data from mitxonline. None of the production runs have this set AFAIK, but we don't want the app trying to query openedx for runs that only exist on edx.org.

Given that I think the current expression makes sense.

Reply:
I see. Ok to keep as is.

Reply:
I had to change expiration_date to use end_date+1 as I run into a validation issue 'Expiration date must be later than end date.' on admin after the import